packages/suite/src/actions/suite/storageActions.ts

```diff
@@ -429,6 +429,7 @@ export const loadStorage = () => async (dispatch: Dispatch, getState: GetState)
                             bridgeDevMode: false, // don't sync bridgeDevMode for now
                         },
                     },
+                    storageLoaded: true,
```

Comment:
do you need this field if it's always set to true? or is there a case where it be false?
catching `STORAGE.LOADED` action in suiteReducer with `draft.storageLoaded = true` looks the same

Reply:
`STORAGE.LOADED` is dispatched also when DB is not available, with `initialState` including `storageLoaded: false`. That way we can detect that no data from DB were actually loaded. The question is whether we want to consider failed loading from DB as `loaded` or not. I dunno.

Reply:
ah, sorry didnt noticed that. resolvig